README.md

```diff
@@ -74,6 +74,12 @@ Your lab directory must include
 * 3pts Tests
 * 2pts Documentation
 
-<!-- links --> 
+<!-- links -->
 [Nodejs http docs]: https://nodejs.org/api/http.html
 [Cowsay docs]: https://github.com/piuccio/cowsay
+
+## Documentation
```

Comment:
Good README description, but go ahead and delete all of the lab instructions so your own README writings can just be at the top.